src/decorators/withCounter/withCounter.jsx

```diff
@@ -10,7 +10,7 @@ type Props = {
   maxLength?: string,
 };
 
-const withCounter = <P: {}>(Component: ComponentType<P>) => {
+const withCounter = <P: any>(Component: ComponentType<P>) => {
```

Comment:
how did you end up fixing this?

Reply:
I think there's no need to specify the `: any` then.

Reply:
Nope could not figure it out juriy